bindings/swift/native/Cargo.toml

```
@@ -0,0 +1,22 @@
+[workspace]
+
+[package]
+name = "iota-wallet-swift-binding"
```

I think this should stay `iota-wallet` as then it'll become the name of the output library, and we don't publish this crate anyway.

```suggestion
name = "iota-wallet"
```